0.3.1: fixture 'pytester' not found

I'm trying to run tests for version 0.3.1 and it fails:

```
$ wget https://files.pythonhosted.org/packages/source/p/pytest_run_parallel/pytest_run_parallel-0.3.1.tar.gz
$ gtar xf pytest_run_parallel-0.3.1.tar.gz
$ cd pytest_run_parallel-0.3.1
$ pytest
============================= test session starts ==============================
platform sunos5 -- Python 3.9.21, pytest-8.3.3, pluggy-1.5.0
rootdir: /tmp/test/pytest_run_parallel-0.3.1
configfile: pyproject.toml
plugins: order-1.3.0, cov-6.1.0
collected 21 items

tests/test_run_parallel.py EEEEEEEEEEEEEEEEEEssE                         [100%]

==================================== ERRORS ====================================
____________________ ERROR at setup of test_default_threads ____________________
file /tmp/test/pytest_run_parallel-0.3.1/tests/test_run_parallel.py, line 21
  def test_default_threads(pytester):
E       fixture 'pytester' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, fail_after_cannot_order, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/tmp/test/pytest_run_parallel-0.3.1/tests/test_run_parallel.py:21
________________________ ERROR at setup of test_marker _________________________
file /tmp/test/pytest_run_parallel-0.3.1/tests/test_run_parallel.py, line 66
  def test_marker(pytester):
E       fixture 'pytester' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, fail_after_cannot_order, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/tmp/test/pytest_run_parallel-0.3.1/tests/test_run_parallel.py:66
____________________ ERROR at setup of test_unittest_compat ____________________
file /tmp/test/pytest_run_parallel-0.3.1/tests/test_run_parallel.py, line 124
  def test_unittest_compat(pytester):
E       fixture 'pytester' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, fail_after_cannot_order, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/tmp/test/pytest_run_parallel-0.3.1/tests/test_run_parallel.py:124

...
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,11 @@
+include tests/conftest.py
+exclude pytest_run_parallel.egg-info
+recursive-exclude * *.egg-info
+recursive-include docs *.bat
+recursive-include docs *.rst
+recursive-include docs Makefile
+include *.lock
+include *.md
+include *.toml
+include *.yaml
+recursive-include docs *.py

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -69,15 +69,22 @@ select = ["E4", "E7", "E9", "F", "I"]
 env_list = ["py38", "py39", "py310", "py311", "py312", "py313", "py313t", "pypy3", "ruff"]
 
 [tool.tox.env_run_base]
-deps = ["pytest>=6.2.0", "pytest-cov", "pytest-order"]
-commands = [[
+deps = ["pytest>=6.2.0", "pytest-cov", "pytest-order", "check-manifest"]
+commands = [
+    [
     "pytest",
     "-v",
     "--cov-report", "lcov",
     "--cov", "src/pytest_run_parallel",
     "--cov", "tests",
     "{posargs:tests}"
-]]
+    ],
+    [
+    "check-manifest",
+    "-u",
+    "-v",
+    ]
+]
 
 [tool.tox.env.ruff]
 skip_install = true

